packages/@contentful--app-scripts/lib/cache-credential/index.test.js

```diff
@@ -0,0 +1,69 @@
+const { cacheEnvVars } = require('./index');
+const fs = require('fs');
+const assert = require('assert');
+
+const fsPromises = fs.promises;
+
+describe('Caching environemnt variables', () => {
+  before(async () => {
+    try {
+      await fsPromises.access('.env', fs.constants.F_OK);
```

Comment:
Maybe you can move the whole test run in a temporary folder or just stub the whole `fs` module and replace it with an in-memory representation for the sake of this test. You can use something like `proxyquire` for that (should be already around)